src/definitions/elements/icon.less

```diff
@@ -372,4 +374,76 @@ i.icons {
   }
 }
 
+& when ((@variationIconGroups) or (@variationIconCorner))
+   and ((@variationIconBordered) or (@variationIconCircular)) {
+  /*************************************************
+     Bordered/circular with corner or group icons
+  *************************************************/
+  i.bordered.icons,
+  i.circular.icons {
+    width: @borderedSize;
+    height: @borderedSize;
+    box-shadow: @borderedShadow;
+    vertical-align: middle;
+  }
+  i.circular.icons {
+    border-radius: 500em;
+  }
+  i.bordered.icons i.icon:first-child,
+  i.circular.icons i.icon:first-child {
+    position: absolute;
+    transform: translateX(-50%) translateY(-50%);
+  }
+
+  & when (@variationIconInverted) {
+    /* Inverted Icon */
+    i.bordered.inverted.icons,
+    i.circular.inverted.icons {
+      border: none;
+      box-shadow: none;
+      background-color: @black;
+      color: @white;
+    }
+  }
+
+  & when (@variationIconCorner) {
+    /* Corner Icon */
+    i.bordered.icons .corner.icon,
+    i.circular.icons .corner.icon {
+      top: auto;
+      left: auto;
+      right: @borderedGroupCornerOffset;
+      bottom: @borderedGroupCornerOffset;
+    }
+    i.bordered.icons .icon.corner[class*="top right"],
+    i.circular.icons .icon.corner[class*="top right"] {
+      top: @borderedGroupCornerOffset;
+      left: auto;
+      right: @borderedGroupCornerOffset;
+      bottom: auto;
+    }
+    i.bordered.icons .icon.corner[class*="top left"],
+    i.circular.icons .icon.corner[class*="top left"] {
+      top: @borderedGroupCornerOffset;
+      left: @borderedGroupCornerOffset;
+      right: auto;
+      bottom: auto;
+    }
+    i.bordered.icons .icon.corner[class*="bottom left"],
+    i.circular.icons .icon.corner[class*="bottom left"] {
+      top: auto;
+      left: @borderedGroupCornerOffset;
+      right: auto;
+      bottom: @borderedGroupCornerOffset;
+    }
+    i.bordered.icons .icon.corner[class*="bottom right"],
+    i.circular.icons .icon.corner[class*="bottom right"] {
+      top: auto;
+      left: auto;
+      right: @borderedGroupCornerOffset;
+      bottom: @borderedGroupCornerOffset;
+    }
+  }
```

Comment:
I think we don't need this, it's the same as line 411 to 417 which is default.

Reply:
This is copied from the existing corner implementation. "bottom right" and default has the same values.

Reply:
🤔 Yeah, but @ko2in is technically right. We could even remove the `bottom right` variant from the usual corner specification.
@fomantic/admins Should we clean this up here or keep the "bottom right" variant nevertheless?

Reply:
If we need to keep `bottom right` variant, we should merge the selectors instead of copy the same definitions.

Reply:
@ko2in Yes, agreed

Reply:
So does this mean I should combine the selectors the default one and the bottom right one?

Reply:
yes, thats the easiest compromise so save CSS declarations and still have the bottom right selector in place.

Reply:
Alright, I will do this later. 🙂

Reply:
Done